dashboard/test/ui/runner.rb

```diff
@@ -9,6 +9,8 @@
 require 'bundler'
 require 'bundler/setup'
 
+require 'cdo/aws/s3'
+require 'cdo/git_utils'
```

Comment:
No reason to expect it would, but in manual runs, the startup time of runner.rb didn't feel qualitatively longer, right?